lib/_http_server.js

```diff
@@ -542,11 +546,12 @@ function Server(options, requestListener) {
   this.httpAllowHalfOpen = false;
 
   this.on('connection', connectionListener);
+  this.on('listening', setupConnectionsTracking, this);
```

Comment:
Where is this third argument being accepted/used?

Reply:
It becomes the `this` of the listener. Which also replies to your other comment. :)

Reply:
I don't think it is needed. `setupConnectionsTracking` will be called with `this` set to the `Server` instance even without this as per `EventEmitter` design, no?

Reply:
> It becomes the `this` of the listener.

But *where* is this done? `EventEmitter`'s `on()` does not support a third argument, unless something is monkey patching it somewhere that I'm overlooking.

Reply:
Yes, I think I confused some with some npm module. XD
Removed.